clustered/integration-test/build.gradle

```diff
@@ -16,11 +16,16 @@
 
 import org.gradle.internal.jvm.Jvm
 
+configurations {
+  serverClasspath
+}
+
 dependencies {
   testCompile project(':dist')
   testCompile project(':clustered:clustered-dist')
   testCompile project(':management')
   testCompile "org.terracotta.management:management-entity-client:$parent.managementVersion"
+  testCompile "org.terracotta.management:monitoring-service-entity:$parent.managementVersion"
```

Comment:
monitoring-service-entity is just only one module that is just for testing purposes: it enabled to consume what is inside the monitoring service. since it is only a few classes, I did not break this test module into client/common/server modules